content/c/c.md

```diff
@@ -0,0 +1,10 @@
+---
+Title: "C"
+Codecademy Hub Page: "https://www.codecademy.com/catalog/language/c"
```

Comment:
For here, let's do `null`:

```
Codecademy Hub Page: null
```

Reply:
Do you still want this inside parenthesis? I left it outside but I can change this.